plugins/vpp/ifplugin/vppcalls/dump_interface_vppcalls.go

```diff
@@ -53,8 +53,14 @@ type InterfaceMeta struct {
 // Dhcp is helper struct for DHCP metadata for interfaces using it
 type Dhcp struct {
 	IfIdx            uint32
+	State            uint8
```

Comment:
The fields from `DhcpLease` should at least come after each other if not as separated struct.

Reply:
Hopefully `Hostname` field will always be same for config and lease./

Reply:
it was as far as I tried

Reply:
Sure.. not sure how it DHCP works in VPP.. but what about dumping dhcp when there is some old dhcp lease still active and some new is being configured or some scenario like that. Might be question for SNK. @lukasmacko

Reply:
ok, let's keep both host names then, so it won't mess up if the lease expires